pages/index.vue

```diff
@@ -90,12 +93,11 @@ export default {
       this.stream.getImageBuffer(this.currFrameTensor);
       this.session.runOp(
         this.operation,
-        this.currFrameTensor,
+        this.frameIndex += 1,
```

Comment:
We need the second argument to be unique since it used for layer caching to speed up long operation chains.